doc/rst/tools/index.rst

```diff
@@ -14,4 +14,5 @@ Contents:
 
    chpldoc/chpldoc
    chplvis/chplvis
-   mason/mason
\ No newline at end of file
+   c2chapel/c2chapel/
```

Comment:
The trailing slash can be removed here, and c2chapel is at the top alphabetically.